packages/p2p/src/pages/my-profile/my-profile-stats/my-profile-name/my-profile-name-business-hours/my-profile-name-business-hours.scss

```diff
@@ -0,0 +1,18 @@
+.my-profile-name-business-hours {
+    .dc-tooltip[data-tooltip]:before {
+        width: max-content;
+        color: var(--text-general);
+        cursor: pointer;
+    }
+
+    &__tooltip {
+        display: flex !important;
```

Comment:
can we avoid using important?

Reply:
i couldn't avoid it sadly, it was messing with the positioning of the toolbox if i dont use !important, but ill check if theres another way

Reply:
by default the display for tooltip is using block, and i need to use !important in this case, since it will misalign the icon and text.

![Screenshot 2024-05-27 at 5 25 44 PM](https://github.com/binary-com/deriv-app/assets/103412909/1491d87a-48b9-4c8f-ae97-8677a7b4b00e)
![Screenshot 2024-05-27 at 5 27 00 PM](https://github.com/binary-com/deriv-app/assets/103412909/fe270bb4-a01f-4252-ad7e-01224f1cb252)